tests/src/JIT/opt/Devirtualization/simple.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+// Some simple interface call devirtualization cases
+
+interface Ix
+{
+    int F();
+}
+
+interface Iy
+{
+    int G();
+}
+
+public class B : Iy, Ix
+{
+    public int F() { return 44; }
+    virtual public int G() { return 55; }
+}
+
+public class Z : B
+{
+    new public int F() { return 33; }
+    override public int G() { return 22; }
+
```

Comment:
You could consider adding an interface method that B explicitly implements (`int Iz.H() { return 11; }`)

You could also consider adding interfaces that B implements and Z re-implements (for various combinations of implementing explicitly/name-matching-virtual/name-matching-nonvirtual/inheriting-from-B).